server/DB/get.js

```diff
@@ -34,6 +34,7 @@ const getUser = async ({ userName }) => {
   }
 }
 
```

Comment:
Not sure what this line does. Hardcoded get request could be fishy. Check with Ian?